visallo/visallo
# Fix Inspector Error for Incomplete Compound Props

Fix an error that occurs when not all of the fields are filled in for a compound property.


Testing Instructions:
1. Create a vertex that has a compound property with not all fields filled in. For example, An address with an empty `street2`.
2. Select the vertex in the UI and ensure that the inspector opens properly.
